.devcontainer/Dockerfile

```diff
@@ -55,3 +55,11 @@ ENV PATH="${PATH}:${SCALA_HOME}/bin"
 ENV DENO_INSTALL="/opt/deno"
 RUN curl -fsSL https://deno.land/x/install/install.sh | sh
 ENV PATH="${DENO_INSTALL}/bin:${PATH}"
+
+# Install .NET SDK 6.0 and dotnet-script
+RUN apt-get update && \
+    apt-get install -y dotnet-sdk-6.0
```

Comment:
I'd just include `dotnet-sdk-6.0` in the long list of apt packages to install above. We can still run `dotnet tool install` and modify the path down here.